src/System.Memory/tests/Memory/Retain.cs

```diff
@@ -45,6 +45,30 @@ public static void MemoryRetainWithPinning()
             handle.Dispose();
         }
 
+        [Fact]
+        public static void MemoryFromEmptyArrayRetainWithPinning()
+        {
+            Memory<int> memory = new int[0];
+            MemoryHandle handle = memory.Retain(pin: true);
```

Comment:
Why are we adding tests for a method that's going away?

Reply:
This is just a reorder to be consistent and makes things easier (avoiding copy/past errors).